server/app.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+
+const app = express();
+
+
+app.use(express.static('public'));
+
+app.get('/calendar/1', (req, res) => {
```

Comment:
I am doing the same to access a single listing, but on the client side (grabbing the [0]th element of the array thats returned by /listinginfo).  
I'm moving on and considering it tech debt for now, but let's stay in touch about what we do here.

Reply:
Here I am querying DB only for the data related to listing 1 instead of bringing all data back to the client-side. We can definitely chat about it.